src/js/menu/menu-button.js

```diff
@@ -37,6 +37,16 @@ class MenuButton extends ClickableComponent {
   }
 
   /**
+   * Hide the menu if the number of items is less than or equal to this threshold. This defaults
```

Comment:
You're right. Maybe just making it be a private instance variable would be easier in the long run.
At least should be a private method.

@justinanastos thoughts?